api/src/api/findTruck.js

```diff
@@ -0,0 +1,127 @@
+const truckMap = require('../api/getTruck');
+const {
+    findLoadById,
+    updateLoad,
+} = require('../models/loadModel');
+const {
+    updateUser,
+} = require('../models/userModel');
+const {
+    findTruck: findTruckByQuery,
+    updateTruck,
+} = require('../models/truckModel');
+const errorHandler = require('../api/errorHandler');
+const {
+    LOAD_STATE,
+    LOAD_STATUS,
+    TRUCK_STATUS,
+} = require('../constants');
+
+
+function findTruck(loadId, res) {
+    findLoadById(loadId)
```

Comment:
Nested `then`s do not look really good. You should try to use `async/await` syntax for this function